src/m365/aad/commands/administrativeunit/administrativeunit-remove.ts

```diff
@@ -0,0 +1,147 @@
+import { AdministrativeUnit } from "@microsoft/microsoft-graph-types";
+import GlobalOptions from "../../../../GlobalOptions.js";
+import { Logger } from "../../../../cli/Logger.js";
+import { validation } from "../../../../utils/validation.js";
+import request, { CliRequestOptions } from "../../../../request.js";
+import GraphCommand from "../../../base/GraphCommand.js";
+import commands from "../../commands.js";
+import { odata } from "../../../../utils/odata.js";
+import { formatting } from "../../../../utils/formatting.js";
+import { Cli } from "../../../../cli/Cli.js";
+
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id?: string;
+  displayName?: string;
+  force?: boolean
+}
+
+class AadAdministrativeUnitRemoveCommand extends GraphCommand {
+  public get name(): string {
+    return commands.ADMINISTRATIVEUNIT_REMOVE;
+  }
+  public get description(): string {
+    return 'Removes an administrative unit';
+  }
+
+  constructor() {
+    super();
+
+    this.#initOptions();
+    this.#initValidators();
+    this.#initOptionSets();
+    this.#initTelemetry();
+    this.#initTypes();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        id: args.options.id !== 'undefined',
+        displayName: args.options.displayName !== 'undefined',
+        force: !!args.options.force
+      });
+    });
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-i, --id [id]'
+      },
+      {
+        option: '-n, --displayName [displayName]'
+      },
+      {
+        option: '-f, --force'
+      }
+    );
+  }
+
+  #initOptionSets(): void {
+    this.optionSets.push(
+      {
+        options: ['id', 'displayName']
+      }
+    );
+  }
+
+  #initValidators(): void {
+    this.validators.push(
+      async (args: CommandArgs) => {
+        if (args.options.id && !validation.isValidGuid(args.options.id)) {
+          return `${args.options.id} is not a valid GUID for option id.`;
+        }
+
+        return true;
+      }
+    );
+  }
+
+  #initTypes(): void {
+    this.types.string.push('id', 'displayName');
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    const removeAdministrativeUnit = async (): Promise<void> => {
+
```

Comment:
Let's remove this superfluous white line

Reply:
Removed